### PR TITLE
fix: Fix multiple typos

### PR DESCRIPTION
**Pull request by minho42**
Fixes typos in comments, string literals, test names, etc.

### CODE REVIEW

File: tests/sentry/api/endpoints/test_system_options.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def test_put_unknown_option(self):
 
     def test_put_simple(self):
         self.login_as(user=self.user, superuser=True)
-        assert options.get("mail.host") != "lolcalhost"
-        response = self.client.put(self.url, {"mail.host": "lolcalhost"})
+        assert options.get("mail.host") != "localhost"
```
Comment (markstory) [inline tests/sentry/api/endpoints/test_system_options.py]: I think these were intentional. This test is now failing.

**Review by BYK — COMMENTED** [review]
Just wow, great job! Once you fix what @markstory pointed out, I think this PR can be merged and we can all learn some English from it 😁

File: tests/sentry/api/endpoints/test_system_options.py (hunk 1 above)
Comment (minho42) [reply tests/sentry/api/endpoints/test_system_options.py]: Thanks @markstory

**Review by markstory — APPROVED** [review]

**Review by tkaemming — APPROVED** [review]
I only looked closely at the things that I'm marked as an owner of (digests) but the rest looks good too at a glance.

**Comment by markstory** [comment]
Looks like there is one conflict now. If we can get that fixed I can get this merged in.

**Comment by minho42** [comment]
Fixed the merge conflicts.
Didn't know how to fix one of them, so kind of ditched my change (src/sentry/static/sentry/app/components/dropdownMenu.jsx).

File: src/sentry/auth/helper.py [hunk 2]
```diff
@@ -466,14 +466,7 @@ def handle_new_user(auth_provider, organization, request, identity):
...
 
-    # If the user has a pending invitation in this organization, we can
-    # immediately assocaite their user.
-    try:
-        member = OrganizationMember.objects.get(organization=organization, email=user.email)
-        member.set_user(user)
-        member.save()
-    except OrganizationMember.DoesNotExist:
-        handle_new_membership(auth_provider, organization, request, auth_identity)
+    handle_new_membership(auth_provider, organization, request, auth_identity)
```
Comment (markstory) [inline src/sentry/auth/helper.py]: This looks like more than just a typo being changed.

**Comment by minho42** [comment]
Sorry @markstory 
I'm not sure how to fix this. Need to figure this out.

File: src/sentry/static/sentry/app/components/dropdownMenu.jsx [hunk 3]
```diff
@@ -1,379 +0,0 @@
-import PropTypes from 'prop-types';
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/dropdownMenu.jsx]: Why is this file in the diff? (It was renamed to be a typescript file)

File: docs-ui/components/form.stories.js [hunk 4]
```diff
@@ -401,7 +401,7 @@ storiesOf('Forms|Fields', module)
   )
   .add(
     'Without a parent Form',
-    withInfo('New form fields used withing having a parent Form')(() => {
+    withInfo('New form fields used without having a parent Form')(() => {
```
Comment (joshuarli) [inline docs-ui/components/form.stories.js]: Wow, that's impressive. What kind of automated typo fixing software are you using @minho42?

Comment (minho42) [reply docs-ui/components/form.stories.js]: Hi @joshuarli,
Thanks for reviewing.
I'm so new to git stuff, and haven't got time to figure out how to resolve the conflicts yet...

Anyway, I'm using my own script to find typos [https://github.com/minho42/typofinder](https://github.com/minho42/typofinder)

**Pull request closed without merging**